packages/core-app-api/src/app/AppManager.tsx

```diff
@@ -484,6 +492,21 @@ export class AppManager implements BackstageApp {
     return this.apiHolder;
   }
 
+  private getIdentitylessApiHolder() {
+    const registrySansIdentity = new ApiFactoryRegistry();
+
+    // Assume all APIs in the current registry are valid.
+    this.apiFactoryRegistry.getAllApis().forEach(apiRef => {
+      const factory = this.apiFactoryRegistry.get(apiRef);
+      // Add all API factories except identity!
+      if (factory && apiRef !== identityApiRef) {
+        registrySansIdentity.register('default', factory);
+      }
+    });
+
+    return new ApiResolver(registrySansIdentity);
```

Comment:
The `ApiResolver` holds the actual API implementations, so by creating another one here, we'll end up with duplicates. That in turn can lead to some pretty unexpected behavior, especially if you for example start some refreshing interval on the sign-in page.